rules/valid-expect-in-promise.js

```diff
@@ -76,12 +75,12 @@ const getFunctionBody = func => {
 };
 
 const getTestFunction = node => {
-  let parent = node.parent;
+  let { parent } = node;
   while (parent) {
     if (isFunction(parent) && isTestFunc(parent.parent)) {
       return parent;
     }
-    parent = parent.parent;
+    ({ parent } = parent);
```

Comment:
This statement is a little confusing - could you help explain what's going on with this destructure statement?

Reply:
Yeah, I just removed it. The syntax is confusing.